Steps4Impact/Notifications/Notifications.swift

```diff
@@ -66,14 +66,14 @@ class NotificationView: UITableViewCell {
   }
 
   func render(_ context: Any) {
-    guard let data = context as? Notification else { return }
+    guard let data = context as? NotificationV1 else { return }
     lblMessage.text = data.message
     // FIXME(compnerd) render this according to the UX style
     lblDate.text = data.date.description
   }
 }
 
-struct Notification {
+struct NotificationV1 {
```

Comment:
The entire reason to have the `V1` suffix is to leave the old type in place.  If you are completely replacing it, just keep the old name.